Formula/pforth.rb

```diff
@@ -0,0 +1,24 @@
+class Pforth < Formula
+  desc "Portable ANS-like Forth written in ANSI C"
+  homepage "http://www.softsynth.com/pforth/"
+  url "https://github.com/philburk/pforth/archive/d71efe1792a1414a5b102ae8e69f3913c70aa8b0.tar.gz"
```

Comment:
This is interesting, why does the upstream dont like to make a release tag for the commit?

Reply:
Hmm should I ask about this in an issue on their repo?

Reply:
Yeah, we require a tagged commit to release

Reply:
Another interesting to call out is the repo does not have OSS license either.